plugins/micromega/coq_micromega.ml

```diff
@@ -38,14 +38,14 @@ let max_depth = max_int
 
 (* Search limit for provers over Q R *)
 let lra_proof_depth =
-  declare_int_option_and_ref ~depr:false ~key:["Lra"; "Depth"] ~value:max_depth
+  declare_int_option_and_ref ~depr:true ~key:["Lra"; "Depth"] ~value:max_depth
```

Comment:
You're not going to mention this and 2 others in the changelog??

Reply:
I do not intend to. They were undocumented and I hope/suspect never used. If I am wrong, users will get the warning.